src/__test__/actions/officeSections.actions.test.js

```diff
@@ -0,0 +1,67 @@
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import constants from '../../_constants';
+
+import loadOfficeSections from '../../_actions/officeSections.actions';
```

Comment:
What's the deal with this filename convention `officeSections.actions`?

Reply:
That's the format the team is using. Do you have any suggestion of how it can be named?